promptsource/templates/aqua_rat/raw/templates.yaml

```
@@ -57,10 +44,12 @@ templates:
 
       '
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: true
+      metrics:
+      - BLEU
+      - Accuracy
```

```suggestion
      metrics:
      - Other
```
Even though it is correct, the way the target is formatted makes that it requires parsing, so the metric can't be applied out of the box.